### PR TITLE
WRP-3081: Increase code coverage for Scroller related modules

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [X] Documentation was verified or is not changed
* [X] UI test was passed or is not needed
* [X] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Increase code coverage for Scroller related modules.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-3081

### CODE REVIEW

File: Scroller/tests/Scroller-specs.js [hunk 1]
```diff
@@ -1,16 +1,43 @@
...
 
 import Scroller from '../Scroller';
 
+const focus = (elm) => fireEvent.focus(elm);
+const keyDownUp = (keyCode) => (elm) => {
+	fireEvent.keyDown(elm, {keyCode});
+	return fireEvent.keyUp(elm, {keyCode});
+};
+
+const pressEnterKey = keyDownUp(14);
```
Comment (daniel-stoian-lgp) [inline Scroller/tests/Scroller-specs.js:14]: the keycode for enter is 13 :). I see the same mistake was made in Sandstone

File: useScroll/tests/ScrollButons-specs.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		});
+	});
+
+	test('should run `focusOnButton`', () => {
+		const functions = renderHook(() => useScrollButtons(props)).result.current;
+		act(() => {
+			functions.focusOnButton();
+		});
+	});
+});
```
Comment (daniel-stoian-lgp) [inline useScroll/tests/ScrollButons-specs.js:63]: these 3 tests don't have any expect. can you check why?

Comment (alexandrumorariu) [reply useScroll/tests/ScrollButons-specs.js]: These functions do not have any tangible results by witch we can construct an expect. I added a meaningless expect for them though.

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**